patcher/src/buttercup/patcher/config.py

```diff
@@ -19,6 +19,7 @@ class ProcessCommand(BaseModel):
     sanitizer: CliPositionalArg[str] = Field(description="Sanitizer")
     crash_input_path: CliPositionalArg[str] = Field(description="Crash Input Path")
     stacktrace_path: CliPositionalArg[str] = Field(description="Stacktrace Path")
+    disable_find_tests: CliImplicitFlag[bool] = Field(default=False, description="Disable find tests")
```

Comment:
I guess another option if you want to get rid of negation is to implement `find_tests` with default value of `True`.

Reply:
Yes indeed, I just realized that pydantic-settings also adds automatically the `--no-<option>` flag, so that should be better!